modules/face/data/mean_shape.txt

```diff
@@ -0,0 +1,195 @@
+137.856,219.266
```

Comment:
To calculate the mean shape we would have to iterate over all the training images' landmarks every time and then scale and centre the mean shape according to the detected bounding box. So I am just preventing the overhead of iterating over images again when detect function is called by storing the mean initially and then scaling that shape according to the face detected each time. So the shape that we get is scaled and centred according to the bounding box detected.